extractors/personal_identifiers/location_extraction/code_snippet_common.md

```diff
@@ -0,0 +1,35 @@
+```python
+import spacy
+from typing import List, Tuple
+
+def location_extraction(text: str, label: str) -> List[Tuple[str, int]]:
```

Comment:
return type currently is List[Tuple[str,int,int]]

Reply:
I think it's unreasonable to return the input as part of the tuple since all will have the same one anyway (so only ram is used without benefit). So i'd change this to -> List[Tuple[int,int]]

Reply:
```suggestion
def location_extraction(text: str) -> List[Tuple[int, int]]:
```

Reply:
if we want we could also return a tuble of three here but then i'd got with <value>, <start>, <end>

Reply:
Not sure how to go about this. We return the <del>label</del> `extraction_keyword ` in all other bricks as well this way. Should we then change this in other bricks, too?

Reply:
I think for common code it's reasonable to return  `<value>, <start_char_idx>, <end_char_idx>` since the spacy doc is lost after the common code so to work with spacy specific information is only possible if the user tokenizes the sentence again. 

If we add information about the return structure in the doc string we should be good for using this snippet without hassle in basically any program.

Two main reasons against:
1. Changes in probably most extraction bricks
2. Not relevant for the cognition release.

Since we already decided that common code and refinery code can have different return values I think we should tackle this to make it more useable outside of refinery. But not now/before the cognition release. So we can either create an issue or note it down somewhere else.

Reply:
Wrote it down here for after the launch: https://github.com/code-kern-ai/bricks/issues/382